app/assets/stylesheets/settings.scss

```diff
@@ -30,3 +30,6 @@ $whitish-grey: #d8d8d8;
 //buttons
 $button-primary: #6abfc3;
 $button-seconday: #cc9543;
+
+//3rd party brands
+$google-color: #dd4b39;
```

Comment:
We will need to check with @adachiu with this color. We could possibly go with googles blue color.